README.md

```diff
@@ -101,3 +101,19 @@ make catalog-build catalog-push
 ```
 
 The Index image powers the listing of the Operator on OperatorHub.
+
+
+## Differences between the GitOps Operator and ArgoCD community Operator 
+
+| Feature | GitOps Operator | Argo CD Community Operator |
+| -------- | -------- | -------- |
+| Out of the box Argo CD Instance     |  ✅    | ❌     |
+| Certified by Red Hat     |   ✅     |   ❌   |
+| Red Hat Productized components     |   ✅     |   ❌   |
+| Environments page for visualizing gitops environments on OpenShift Console   |   ✅     |   ❌   |
+| OpenShift Cluster Configuration using Argo CD    |   ✅     |   ❌  |
+| Support for Red Hat Single Sign-on     |   ✅     |   Supports Keycloak   | 
+| Support for Redis 6     |  Supports Redis 5     |   ✅    |
+| Support for Argo CD exports     |  ❌     |   ✅    |
+| Supports Single Namespace Installation     |  ❌     |   ✅    |
+| Support for Kubernetes     |  ❌     |   ✅    |
```

Comment:
OpenShift specific features: Route, RHSSO, Auto TLS, DeploymentConfig
Differences in terms of cluster config, RBAC
Kam , backend service
Export/Import tools
Secure comm between Redis server/client

Reply:
@wtam2018 sorry but I did not understand.
what is `OpenShift specific features: Route, RHSSO, Auto TLS, DeploymentConfig` ? All of them are supported by both community(openshift) operator and gitops operator.

RBAC ?

I already mentioned about Export, are you talking about something else ?

Secure comm between Redis server/client ? - I did not get this.

Reply:
@iam-veeramalla Sorry, I quickly jotted down my thought. If and when OpenShift APIs dependency is removed from upstream, these features will be supported by GitOps Operator only. ATM, you are right. They can be found in both operators 
By RBAC, I mean Policy Rules. When cluster configuration is enabled, policy rules/roles applied to the SA of the application controller are different between the two operators.
Yes, you have already mentioned Export.
Downstream Redis server currently does not support secure connections I believe..
@jopit @chetan-rns @iam-veeramalla PTAL at the updated table.